README.md

```diff
@@ -14,6 +14,9 @@ The script utilises the following environment variables.
 | AWS_REGION            | Yes      | -             | AWS region to connect to                                                    |
 | MIN_SEVERITY          | No       | `ALL`         | Sets the minimum vulnerability severity level to indicate a failure         |
 | MAX_WAIT              | No       | `120`         | Sets how long, in seconds, to wait for ECR scan results to become available |
+| SLACK_DISABLE         | No       | -             | If set and non-empty, will disable sending of Slack reports                 |
```

Comment:
Is `SLACK_DISABLE` needed? Can we not infer this from the presence or absence of `SLACK_CHANNEL` and `SLACK_WEBHOOK`?